corehq/apps/userreports/util.py

```diff
@@ -191,3 +188,28 @@ def get_ucr_class_name(id):
 
 def get_async_indicator_modify_lock_key(doc_id):
     return 'async_indicator_save-{}'.format(doc_id)
+
+
+def copy_static_reports(from_domain, to_domain, report_map):
```

Comment:
Also the name is a bit weird since it's not copying anything. How about `get_static_report_mapping`